src/subdomains/supporting/payment/dto/fee.dto.ts

```diff
@@ -1,4 +1,5 @@
 export class FeeDto {
+  feeIds: string; // Semicolon separated id's
```

Comment:
There are a lot of semicolons all over the place 😄  maybe we should keep it as a number array in the DTO and only map it to string in the respective entities?